src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    private int radius;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        setColor(color);
+    }
+
+    public Circle() {
+    }
+
+    public Circle getRandomCircle() {
+        String color = colorSupplier.getColorOfFigure();
+        int radius = new Random().nextInt(15);
+        return new Circle(radius, color);
+    }
+
+    public Circle getDefaultCircle() {
+        String color = Color.WHITE.name();
```

Comment:
delete this methods

Reply:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/161030105/280f1ca1-9f07-42ac-8163-138f014d1f28)

Should these be methods for different figures in the FigureSupplier class or in classes for those figures?